components/builder-originsrv/src/data_store.rs

```diff
@@ -381,6 +381,28 @@ impl DataStore {
         if rows.len() != 0 { Ok(true) } else { Ok(false) }
     }
 
+    pub fn my_origins(
+        &self,
+        mor: &originsrv::MyOriginsRequest,
+    ) -> SrvResult<originsrv::MyOriginsResponse> {
+        let conn = self.pool.get(mor)?;
+        let rows = &conn.query(
+            "SELECT * FROM my_origins_v1($1) ORDER BY name",
```

Comment:
Can the ORDER BY be in the sproc?

Reply:
There is one in the sproc, but since this function queries across shards, it needs to be ordered outside of the proc in order for the list to actually be in order.  The ORDER BY inside the proc only applies per shard (and since we're only pulling 1 origin per shard it's actually irrelevant).